packages/deployment/upgrade-test/upgrade-test-scripts/coreUpgradeHelpers.mjs

```diff
@@ -131,3 +132,212 @@ export const executeOffer = async (address, offerPromise) => {
     '--keyring-backend=test',
   );
 };
+
+export const getUser = async user => {
+  return agd.keys('show', user, '-a', '--keyring-backend=test');
+};
+
+export const openVault = (address, mint, collateral) => {
```

Comment:
The vault helpers aren't about coreUpgrade. Good to have them in a utility module but please make a different one. E.c. `econHelpers.mjs` or `interProtocol.js`.